package.json

```diff
@@ -60,6 +60,7 @@
   "dependencies": {
     "@azure/functions": "^1.2.2",
     "axios": "^0.19.2",
-    "http-status-codes": "^2.1.4"
+    "http-status-codes": "^2.1.4",
+    "nock": "^13.1.2"
```

Comment:
From looking at what nock is, it seems to be a tool for HTTP Server Mocking. If so, this should be a dev dependency